feature/experimental/bgp/otg_tests/bgp_2byte_4byte_asn_policy_test/bgp_2byte_4byte_asn_policy_test.go

```diff
@@ -0,0 +1,565 @@
+	if peerAS != nbr.peerAS {
+		t.Errorf("BGP peerAs: got %v, want %v", peerAS, nbr.peerAS)
+	}
+
+	// Check BGP localAS from telemetry.
+	localAS := gnmi.Get(t, dut, nbrPath.State()).GetLocalAs()
+	if localAS != nbr.localAS {
+		t.Errorf("BGP localAS: got %v, want %v", localAS, nbr.localAS)
+	}
+
+	// Check BGP globalAS from telemetry.
+	globalAS := gnmi.Get(t, dut, glblPath.State()).GetAs()
+	if globalAS != globalAsNumber {
+		t.Errorf("BGP globalAS: got %v, want %v", globalAS, nbr.localAS)
+	}
+
+	verifyPrefixesTelemetry(t, dut, 3, nbr.isV4)
+}
+
+func configureATE(t *testing.T, ateParams *bgpNbr, connectionType string, prefixes []string, asWidth int) gosnappi.Config {
+	t.Helper()
+	t.Log("Create otg configuration")
+	ate := ondatra.ATE(t, "ate")
+	port1 := ate.Port(t, "port1")
+	config := gosnappi.NewConfig()
+
+	config.Ports().Add().SetName(port1.ID())
+	srcDev := config.Devices().Add().SetName(ateSrc.Name)
+	srcEth := srcDev.Ethernets().Add().SetName(ateSrc.Name + ".Eth").SetMac(ateSrc.MAC)
+	srcEth.Connection().SetPortName(port1.ID())
+	srcIpv4 := srcEth.Ipv4Addresses().Add().SetName(ateSrc.Name + ".IPv4")
+	srcIpv4.SetAddress(ateSrc.IPv4).SetGateway(dutSrc.IPv4).SetPrefix(uint32(ateSrc.IPv4Len))
+	srcIpv6 := srcEth.Ipv6Addresses().Add().SetName(ateSrc.Name + ".IPv6")
+	srcIpv6.SetAddress(ateSrc.IPv6).SetGateway(dutSrc.IPv6).SetPrefix(uint32(ateSrc.IPv6Len))
+
+	srcBgp := srcDev.Bgp().SetRouterId(srcIpv4.Address())
+	if ateParams.isV4 {
+		srcBgpPeer := srcBgp.Ipv4Interfaces().Add().SetIpv4Name(srcIpv4.Name()).Peers().Add().SetName(ateSrc.Name + ".BGP4.peer")
+		srcBgpPeer.SetPeerAddress(ateParams.peerIP).SetAsNumber(ateParams.localAS)
+		if connectionType == connInternal {
+			srcBgpPeer.SetAsType(gosnappi.BgpV4PeerAsType.IBGP)
+		} else {
+			srcBgpPeer.SetAsType(gosnappi.BgpV4PeerAsType.EBGP)
+		}
+		if asWidth == 2 {
+			srcBgpPeer.SetAsNumberWidth(gosnappi.BgpV4PeerAsNumberWidth.TWO)
+		}
+		subnetAddr1, subnetLen1 := prefixAndLen(prefixV4[0])
+		subnetAddr2, subnetLen2 := prefixAndLen(prefixV4[1])
+		subnetAddr3, subnetLen3 := prefixAndLen(prefixV4[2])
+
+		network1 := srcBgpPeer.V4Routes().Add().SetName("bgpNeti1")
+		network1.SetNextHopIpv4Address(ateSrc.IPv4).
+			SetNextHopAddressType(gosnappi.BgpV4RouteRangeNextHopAddressType.IPV4).
+			SetNextHopMode(gosnappi.BgpV4RouteRangeNextHopMode.MANUAL)
+		network1.Addresses().Add().SetAddress(subnetAddr1).SetPrefix(subnetLen1).SetCount(1)
+		network1.AsPath().Segments().Add().SetAsNumbers([]uint32{55000, 4400, 3300})
+
+		network2 := srcBgpPeer.V4Routes().Add().SetName("bgpNeti2")
+		network2.SetNextHopIpv4Address(ateSrc.IPv4).
+			SetNextHopAddressType(gosnappi.BgpV4RouteRangeNextHopAddressType.IPV4).
+			SetNextHopMode(gosnappi.BgpV4RouteRangeNextHopMode.MANUAL)
+		network2.Addresses().Add().SetAddress(subnetAddr2).SetPrefix(subnetLen2).SetCount(1)
+		network2.AsPath().Segments().Add().SetAsNumbers([]uint32{55000, 7700})
+		network2.Communities().Add().SetAsNumber(200).SetAsCustom(1)
```

Comment:
@octpetre : Looks like we need to add SetType(gosnappi.BgpCommunityType.MANUAL_AS_NUMBER) to make that explicit. (I'm not sure what OTG is setting as a default here, but the test works after that change.)

Reply:
Yes, the model doesn't define a default value...so I'm adding the type